main.js

```diff
@@ -73,14 +81,28 @@ function main() {
         var name = workshopData[i][COLUMN_WORKSHOP_NAME];
         var number = i;
         var capacity = workshopData[i][COLUMN_WORKSHOP_CAPACITY];
+        var location = workshopData[i][COLUMN_WORKSHOP_BUILDING] + " " + workshopData[i][COLUMN_WORKSHOP_ROOM];
+        
+        //Checks name, number, and capacity, for valid inputs.//
```

Comment:
This says it checks workshop number, but I only see name and capacity. Should there be something to check the number as well?

Reply:
the number will always be an int since it will be so I never saw the need to check i.